### PR TITLE
Remove node_modules from being served by Jekyll

### PR DESCRIPTION
**Pull request by jszwedko**
Instead, use a Makefile to copy the needed assets into their respective
vendor directories. This allows us to avoid having Jekyll parse and copy
over everything in node_modules which we don't really want to serve
anyway.

In the future, this could also let us stop checking in `node_modules` altogether if it was part of the build process.

@jasonlally what do you think of this approach?

It was largely built on https://simpleit.rocks/how-to-add-bootstrap-4-to-jekyll-the-right-way/

### CODE REVIEW

**Comment by jasonlally** [comment]
Yeah, I agree, I'd rather not check in node_modules.

I added some js dependencies for the projects update, so I'll probably need to update the makefile to copy those over.

**Comment by jasonlally** [comment]
@jszwedko - I merged changes from gh-pages and updated the makefile to include new libraries for the projects page.

I also added a line to copy over the font-awesome font files to a folder within css.

I think this is okay to merge, but let me know if you have anything else. Also, we could probably drop node_modules from committing without any issue now, but maybe we do that on a separate branch?

File: README.md [hunk 1]
```diff
@@ -12,11 +12,13 @@ Heavily inspired by [Chi Hack Night's website](https://chihacknight.org/).
...
   * Usage of [rbenv](https://github.com/rbenv/rbenv) recommended
-* Run `bundle`
-* `bundle exec jekyll serve`
+* Run `make`
 
 [yarn](https://yarnpkg.com/lang/en/docs/install/) is used for managing external asset libraries (like Bootstrap, jQuery,
-etc.). Please install and use this if you are adding a new external resource.
+etc.).
+
+If you add a new external resource, be sure to add lines to the relevant [`Makefile`](Makefile) target to copy
```
Comment (jszwedko) [inline README.md:20]: :+1:

**Review by jszwedko — COMMENTED** [review]
Awesome! We can keep iterating on this, but I think this is a definite step forward.

**Comment by jszwedko** [comment]
Removed in d53c17828592d0ddd6e6ce001298a7ce4a57d6e1

**Pull request merged**